sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIAsyncClient.java

```diff
@@ -423,7 +423,6 @@ public Flux<Completions> getCompletionsStream(String deploymentOrModelName, Comp
      * @return chat completions for the provided chat messages. Completions support a wide variety of tasks and generate
      *     text that continues from or "completes" provided prompt data on successful completion of {@link Mono}.
      */
-    @Generated
```

Comment:
Dev need to maintain this 2 APIs, as

```
Convenience API is not generated, as operation 'getChatCompletions' refers Union 'FunctionCallPreset | FunctionName'
```

Reply:
I am not entirely sure we want to remove these annotations from these convenience methods. Could we simply update the version of autorest in this PR and then we evaluate how we handle methods exposing types with `union` on our side?

Reply:
```
Convenience API is not generated, as operation 'getChatCompletions' refers Union 'FunctionCallPreset | FunctionName'
```

Was this a side effect of updating the autorest version?

Reply:
Srikanta explained this change in behaviour offline to me! Sorry for the confusion. Will merge accordingly.